frontends/web/src/api/subscribe.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2021 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { apiSubscribe, Event, Unsubscribe } from '../utils/event';
+import { apiGet } from '../utils/request';
+
+export function subscribeEndpoint(endpoint: string, cb): Unsubscribe {
+    return apiSubscribe(endpoint, (event: Event) => {
+        switch (event.action) {
+        case 'replace':
+            cb(event.object);
+            break;
+        case 'reload':
+            apiGet(event.subject).then(object => cb(object)).catch(console.error);
+            break;
+        default:
+            console.warn('need to support', event);
```

Comment:
Throw and error since there are no other event actions... or are there?

Reply:
I deprecated 3 actions, afaik they have never been used.